### PR TITLE
Actually Finishing the gun cells

### PR DESCRIPTION
**Pull request by Baldir-Odensen**
## About The Pull Request

In febuary i made the Weapon gun cells. Now i have decided to finish it. the weapon gun cells will now fit in the secbelt and webbing, not fit in the RPED and i moved the cells from the power section to the weapon section to be with all the other magazines.

## Why It's Good For The Game

this was the original intention behind the guncells. you can no longer carry 100 guncells in a bluespace RPED but they instead fit in the secbelt. this will make them more useable by security and posibly more user friendly.

## Changelog
:cl:
del: Removed the RPED is now unable to accept weapon cells.
tweak: The secbelt has been modified to take weapon cells,
tweak: the gun cell design has been moved from power to weapons.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -221,7 +221,8 @@
 		/obj/item/radio,
 		/obj/item/clothing/gloves,
 		/obj/item/restraints/legcuffs/bola,
-		/obj/item/holosign_creator/security
+		/obj/item/holosign_creator/security,
+		/obj/item/stock_parts/cell/gun   //waspstation eddit Gun cells fit where they should and not where they dont
```
Comment (MarkSuckerberg) [inline code/game/objects/items/storage/belt.dm:225]:
```suggestion
		/obj/item/stock_parts/cell/gun   //Waspstation edit Gun cells fit where they should and not where they dont
```
nitpick

File: code/modules/research/designs/weapon_designs.dm [hunk 2]
```diff
@@ -373,3 +373,25 @@
 	build_path = /obj/item/melee/baton/boomerang
 	category = list("Weapons")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
```
Comment (MarkSuckerberg) [inline code/modules/research/designs/weapon_designs.dm:376]:
```suggestion

//Wasp Edit - Energy Gun power cells
```

File: code/modules/research/designs/weapon_designs.dm [hunk 3]
```diff
@@ -373,3 +373,25 @@
...
+/datum/design/gun_cell/upgraded
+	name = "Upgraded Weapon Power Cell"
+	desc = "A upgraded power cell for weapons holds 20 MJ of energy."
+	id = "gun_cell_upgraded"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron = 700, /datum/material/gold = 150, /datum/material/silver = 150, /datum/material/glass = 80)
+	construction_time=100
+	build_path = /obj/item/stock_parts/cell/gun/upgraded/empty
+	category = list("Misc","Weapons")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_SECURITY
```
Comment (MarkSuckerberg) [inline code/modules/research/designs/weapon_designs.dm:397]:
```suggestion
	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_SECURITY
//Wasp end
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by Baldir-Odensen** [comment]
ok i think i have done the things now

**Review by MarkSuckerberg — APPROVED** [review]
👍

**Comment by Maldaris** [comment]
This is a fix for an existing feature, this shouldn't require a vote.

**Pull request merged**